spec/javascripts/BowlingGameSpec.js

```diff
@@ -0,0 +1,70 @@
+describe('Bowling', function() {
+  it('Gutter game', function() {
+    const bowling = new Bowling();
```

Comment:
If you can, try to avoid constants. I can see totally why it's necessary however it lowers how 'changeable' your code is

Reply:
Hold on, ignore that! forgot this was a test file, if anything that's smart af